lib/miq_automation_engine/engine/miq_ae_state_machine.rb

```diff
@@ -74,7 +74,7 @@ def process_state(f, message, args)
 
         # Check the ae_result and set the next state appropriately
         if   @workspace.root['ae_result'] == 'ok'
-          $miq_ae_logger.warn "Processed State =[#{f['name']}]"
+          $miq_ae_logger.info "Processed State =[#{f['name']}]"
         elsif @workspace.root['ae_result'] == 'skip'
           $miq_ae_logger.warn "Skipping State =[#{f['name']}]"
```

Comment:
@matthewd 
If something is skipped it should be a warning because in a normal flow we would not skip. If there is an automate method that caused a skip, the warning should help us locate it.